app/src/main/java/com/deadlineshooters/yudemy/fragments/AccountFragment.kt

```diff
@@ -72,22 +76,25 @@ class AccountFragment : Fragment() {
         editProfile = view.findViewById(R.id.editProfile)
         editImage = view.findViewById(R.id.editImage)
 
-        val currentActivity = requireActivity()
+        email.text = curUserEmail.toString()
+
         if (requireActivity() is InstructorMainActivity) {
-            navigateIns.setText("Switch to Student View")
+            navigateIns.text = "Switch to Student View"
         } else {
-            navigateIns.setText("Switch to Instructor View")
+            navigateIns.text = "Switch to Instructor View"
         }
 
         navigateIns.setOnClickListener {
```

Comment:
thay nguyên đoạn này thành này nha: Tại hiện tại lúc từ Instructor mà qua student ko được 
 val currentActivity = requireActivity()
        navigateIns.setOnClickListener {
            if (currentActivity is InstructorMainActivity) {
                startActivity(Intent(currentActivity, StudentMainActivity::class.java))
            } else {
                startActivity(Intent(currentActivity, InstructorMainActivity::class.java))
            }
        }

Reply:
ủa được mà bạn